src/i18ndude/tests/test_untranslated.py

```diff
@@ -138,6 +138,26 @@ def test_find_untranslated_aria_label_attribute(self):
             'lacks i18n:attributes',
             result_with_translated_aria_label)
 
+    def test_find_untranslated_disallows_nonword_chars_in_i18n_name(self):
+        result_with_space_in_name = find_untranslated(
+            '<div i18n:translate="">Something <span i18n:name="xfoo xbar">amiss</span></div>')
+        self.assertIn(
+            'invalid non-word characters (space, punctuation) in i18n:name',
```

Comment:
I suggest defining this error message in a variable, since you are using the same message several times.